.github/workflows/e2e.yaml

```diff
@@ -0,0 +1,150 @@
+---
+name: E2E
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, labeled, unlabeled]
+env:
+  TOOLS_PATH: "/opt/tools/bin"
+jobs:
+  # Only the following E2E tests require EKS:
+  # - test/e2e
+  # - test/e2e-core
+  # - test/e2e-db-upgrade
+  # - test/e2e-operator-upgrade
+  # TODO: Review this condition once we have more tests running on K3D.
+  e2e-eks-tests:
+    name: EKS tests
+    if: |
+      contains(github.event.pull_request.labels.*.name, 'test/e2e') ||
+      contains(github.event.pull_request.labels.*.name, 'test/e2e-operator-upgrade') ||
+      contains(github.event.pull_request.labels.*.name, 'test/e2e-core') ||
+      contains(github.event.pull_request.labels.*.name, 'test/e2e-db-upgrade')
+    timeout-minutes: 80
+    concurrency: eks-e2e-tests
+    strategy:
+      fail-fast: true
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v4
+
+      - name: Set up Go release
+        uses: actions/setup-go@v5
+        with:
+          go-version-file: 'go.mod'
+          cache: true
+
+      - name: Build operator
+        run:
+          make build
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Install Kubernetes tools
+        uses: ./.github/actions/install-kubernetes-tools
+        with:
+          install-kubectl: true
+          install-kuttl: true
+          install-k3d: false
+
+      - name: Install EKS tools
+        run: |
+          # Install EKSCTL
+          ARCH=amd64
+          PLATFORM=$(uname -s)_${ARCH}
+          curl -sLO "https://github.com/eksctl-io/eksctl/releases/latest/download/eksctl_${PLATFORM}.tar.gz"
+          tar -xzf eksctl_${PLATFORM}.tar.gz -C "${TOOLS_PATH}"
+          chmod +x "${TOOLS_PATH}"/eksctl
+
+      - name: Configure AWS
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          aws-access-key-id: ${{ secrets.EVEREST_EKS_E2E_IAM_USER_ACCESS_KEY }}
+          aws-secret-access-key: ${{ secrets.EVEREST_EKS_E2E_IAM_USER_ACCESS_KEY_SECRET }}
+          aws-region: us-east-2
+
+      - name: Update EKS kubeconfig
+        run: |
+          aws eks update-kubeconfig --name everest-e2e-dev-eks-cluster
+
+      - name: Scale up cluster
+#        if: |
+#          contains(github.event.pull_request.labels.*.name, 'test/e2e') ||
+#          contains(github.event.pull_request.labels.*.name, 'test/e2e-core') ||
+#          contains(github.event.pull_request.labels.*.name, 'test/e2e-db-upgrade') ||
+#          contains(github.event.pull_request.labels.*.name, 'test/e2e-operator-upgrade')
```

Comment:
Since this job is already protected by this condition we don't need it and there's no point in keeping it commented either.
```suggestion
```